File: Iniciante/1009 - Salário com Bônus/Python/1009.py
def main():
    seller_name = input()
    fixed_salary = float(input())
    total_sales_month = float(input())
    COMMISSION_RATE = 0.15
    total_salary = (total_sales_month * COMMISSION_RATE) + fixed_salary
    print(f"TOTAL = R$ {total_salary:.2f}")


if __name__ == "__main__":
    main()

File: Iniciante/1009 - Salário com Bônus/JavaScript/1009.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const lines = input.split("\n");
  const sellerName = lines.shift();
  const fixedSalary = parseFloat(lines.shift());
  const totalSalesMonth = parseFloat(lines.shift());
  const COMMISSION_RATE = 0.15;
  const totalSalary = totalSalesMonth * COMMISSION_RATE + fixedSalary;
  console.log(`TOTAL = R$ ${totalSalary.toFixed(2)}`);
}

main();
